docs/developer-documentation.md

```diff
@@ -120,8 +120,40 @@ Provides the following factory methods:
   - `size`: file size in bytes.
   - `title` (optional): file title, defaults to file `name` if empty.
   - `language_code` (optional): two letter language code of the current file's language.
-- `FileValueObject::fromFileEntity(FileInterface $file_entity)`: accept an object implementing the
+
+- `FileValueObject::fromFileEntity(FileInterface $file_entity)`: accepts an object implementing the
   `\Drupal\file\FileInterface` interface.
 
+#### `GalleryItemValueObject`
+
+Used in the following patterns:
+
+- [`gallery`](../templates/patterns/gallery/gallery.ui_patterns.yml)
+
+Provides the following factory methods:
+
+- `DateValueObject::fromArray(array $values = [])`: accepts an array with the following properties:
+  - `thumbnail`: Thumbnail to be rendered on the gallery item.
+  - `caption`: Caption for the gallery item.
+  - `classes`: Extra classes for the gallery item.
+  - `icon`: Icon for the gallery item.
+
+#### `ImageValueObject`
+
+Used in the following patterns:
+
+- [`gallery`](../templates/patterns/gallery/gallery.ui_patterns.yml)
+
+Provides the following factory methods:
+
+- `DateValueObject::fromArray(array $values = [])`: accepts an array with the following properties:
```

Comment:
This should be `ImageValueObject` not `DateValueObject`.

Reply:
:+1: